prac_05/wimbledon.py

```diff
@@ -0,0 +1,48 @@
+"""
+Wimbledon
+Estimate: 40 minutes
+Actual:   39 minutes
+"""
+FILENAME = "wimbledon.csv"
+
+def main():
+
+    records = get_records(FILENAME)
+    champion_to_count, countries = process_records(records)
+    display_results(champion_to_count, countries)
+
+
+def process_records(records):
+
```

Comment:
It would be good if the function formatting is consistent throughout the program. I noticed that the main() and process_records() functions have an empty line below but not in display_records() and get_records()